packages/rocketchat-api/server/default/info.js

```diff
@@ -1,5 +1,16 @@
 RocketChat.API.default.addRoute('info', { authRequired: false }, {
 	get: function() {
-		return RocketChat.Info;
+		const user = this.getLoggedInUser();
+
+		if (user && RocketChat.authz.hasRole(user._id, 'admin')) {
+			return RocketChat.API.v1.success({
+				info: RocketChat.Info
+			});
+		}
+
+		return RocketChat.API.v1.success({
+			version: RocketChat.Info.version,
+			build: RocketChat.Info.build
```

Comment:
As the system build information is usually information which system administrators want to keep private, let's remove this property from here and you'll need to remove it from the tests.